releasetasks/templates/enUS.yml.tmpl

```diff
@@ -305,6 +307,8 @@
         retries: 5
         routes:
             - tc-treeherder-stage.{{ branch }}.{{ revision_hash }}
+            - index.releases.{{ branch }}.{{ revision }}.{{ product }}.{{ version | replace(".", "_") }}.build{{ buildNumber }}.partials_beetmove.{{ partial_version | replace(".", "_") }}.{{ platform }}.{{ locale }}
```

Comment:
nit: for the completes en-US and l10n you use 'beetmover'. maybe use 'partial_beetmover' here?